vscode-dotnet-runtime-library/src/Acquisition/AcquisitionInvoker.ts

```diff
@@ -33,7 +33,8 @@ export class AcquisitionInvoker extends IAcquisitionInvoker {
         const installCommand = await this.getInstallCommand(installContext.version, installContext.installDir);
         return new Promise<void>((resolve, reject) => {
             try {
-                cp.exec(winOS ? `powershell.exe -ExecutionPolicy unrestricted -File ${installCommand}` : installCommand,
+                const windowsFullCommand = `powershell.exe -ExecutionPolicy unrestricted -Command "& { [Net.ServicePointManager]::SecurityProtocol = 'Tls12, Tls13' ; & ${installCommand} }`;

```

Comment:
Is it ok toe run this comment several time? What if the install script itself has TLS set already?

Reply:
Moved to and answered at https://github.com/dotnet/vscode-dotnet-runtime/pull/60#discussion_r413871656